src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1340,11 +1340,11 @@ public void LanguageVersionAdded_Canary()
         {
             // When a new version is added, this test will break. This list must be checked:
             // - update the "UpgradeProject" codefixer
-            // - update the IDE drop-down for selecting Language Version
+            // - update the IDE drop-down for selecting Language Version (in project-systems repo)
```

Comment:
Where is this work being tracked? This gets forgotten every release, and I get to laugh at somebody. I'd like to stop doing that actually.